### PR TITLE
feat: bump angular 17

### PR DESCRIPTION
**Pull request by damingerdai**
related: #261

### CODE REVIEW

**Comment by damingerdai** [comment]
image

**Comment by damingerdai** [comment]
> image

fix it by [fix(multiple): ensure all public theming mixins are exposed at top-level](https://github.com/ng-matero/extensions/issues/261)

**Comment by LanderBeeuwsaert** [comment]
@damingerdai we would want to upgrade to angular 17 already but are blocked by this.
Is it possible to somehow install your branch and use that?
I've tried with different variations of:
"@ng-matero/extensions": "github:@ng-matero/extensions.git#damingerdai:feat/261",
and even tried cloning your forked repository instead but I don't get it working. Possibly I'm doing something wrong?
Would be great if you can give me a pointer!

**Comment by damingerdai** [comment]
@nzbin please review

File: package.json [hunk 1]
```diff
@@ -34,19 +34,19 @@
...
+    "@angular/core": "^17.0.2",
+    "@angular/forms": "^17.0.2",
+    "@angular/material": "^17.0.0",
+    "@angular/material-date-fns-adapter": "^17.0.0",
+    "@angular/material-luxon-adapter": "^17.0.0",
+    "@angular/material-moment-adapter": "^17.0.0",
+    "@angular/platform-browser": "^17.0.2",
+    "@angular/platform-browser-dynamic": "^17.0.2",
+    "@angular/router": "^17.0.2",
     "@ng-select/ng-select": "^11.1.0",
```
Comment (A77AY) [inline package.json:50]: Need to update to 12

**Review by A77AY — CHANGES_REQUESTED** [review]

**Pull request merged**